login-workflow/src/screens/Login.tsx

```diff
@@ -104,34 +104,48 @@ export const Login: React.FC = () => {
     const { t } = useLanguageLocale();
     const authUIActions = useAccountUIActions();
     const authUIState = useAccountUIState();
-    const authProps = useInjectedUIContext();
+    const {
+        showRememberMe = true,
+        allowDebugMode = false,
+        showSelfRegistration = true,
+        enableResetPassword = true,
+        showContactSupport = true,
+        showCybersecurityBadge = true,
+        projectImage,
+        loginFooter,
+        loginHeader,
+    } = useInjectedUIContext();
     const { routes } = useRoutes();
     const theme = useTheme();
     const classes = useStyles();
 
     const passwordField = useRef<any>(null);
 
     // Local State
-    const [rememberPassword, setRememberPassword] = React.useState(securityState.rememberMeDetails.rememberMe ?? false);
-    const [emailInput, setEmailInput] = React.useState(securityState.rememberMeDetails.email ?? '');
+    const [rememberPassword, setRememberPassword] = React.useState(
+        showRememberMe ? securityState.rememberMeDetails.rememberMe ?? false : false
```

Comment:
What does the '??' do compared to the '?' ?

Reply:
`??` does a "if the value to my left is defined, use it, otherwise use the value to my right".

The official name is the "nullish coalescing operator": https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Without that it would look like:
```
showRememberMe ? (securityState.rememberMeDetails.rememberMe ? securityState.rememberMeDetails.rememberMe : false) : false
```

Reply:
It's useful when you're dealing with _default_ values

Reply:
The change on this line is basically just adding another conditional check...if showRememberMe is false, we always use false for the value, regardless of whether we have a value in the security state or not.